### PR TITLE
CKKH Chitter Challenge 

### PR DESCRIPTION
**Pull request by CKKH**
One Unit test is failing, if ran in isolation it works:

rspec ./spec/user_spec.rb:30 # User #authenticate Sign in if username and password are correct

### CODE REVIEW

File: spec/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      expect(User.authenticate('christos', 'Makers123')).to eq(user)
+    end
+  end
+
+  context '#valid?' do
+    it 'confirms uniqueness of unused email on signup' do
+      expect(user.valid?).to eq true
+    end
+
+    it 'Returns false if email of existing user used on signup' do
```
Comment (CazaBelle) [inline spec/user_spec.rb]: Good tests for User class

File: spec/chirrup_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require 'chirrup'
```
Comment (CazaBelle) [inline spec/chirrup_spec.rb]: Might be worth having a test for the time that the chirrup was created? Look into Timecop and time.freeze

File: Gemfile [hunk 3]
```diff
@@ -1,12 +1,20 @@
 source 'https://rubygems.org'
```
[the remark this answers is not in the document]
Comment (CKKH) [reply Gemfile]: Best feedback of my coding career, thanks!

File: spec/features/chirrup_feature_spec.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+require_relative 'web_helpers'
```
[the remark this answers is not in the document]
Comment (CKKH) [reply spec/features/chirrup_feature_spec.rb]: Would make more sense to move that into spec_helper now you mention it, thanks.

File: spec/features/delete_account_feature_spec.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+require_relative 'web_helpers'
+
+RSpec.feature 'Delete account' do
+  context 'Chosing whether to delete account or not' do
```
[the remark this answers is not in the document]
Comment (CKKH) [reply spec/features/delete_account_feature_spec.rb]: Nice, I'll make use of that to DRY up my features, thanks Caz.

File: spec/chirrup_spec.rb (hunk 2 above)
Comment (CKKH) [reply spec/chirrup_spec.rb]: This would be a good way to make my tests my comprehensive, will do!

**Pull request closed without merging**